cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,81 @@
+import { generateUser } from '../support/commands';
+
 /// <reference types='cypress' />
 
+const user = generateUser();
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('');
+  });
+
+  it(' should be elements of pagination is present', () => {
+    cy.get('.-pageInfo').should('exist');
+    cy.get('.-previous > .-btn').should('exist');
+    cy.get('.-pageJump > input').should('exist');
+    cy.get('.-totalPages').should('exist');
+    cy.get('.-next > .-btn').should('exist');
+  });
+
+  it(' should add new user', () => {
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.userEmail);
```

Comment:
The property 'userEmail' is used here, but in the assertion on line 33, 'user.email' is used. Ensure consistency in property names.